packages/web/src/components/tipping/support/SupportingList.tsx

```diff
@@ -47,7 +47,7 @@ const SupportingListForProfile = ({ profile }: { profile: User }) => {
     }
   }, [profile, dispatch])
 
-  return profile && rankedSupportingList.length > 0 ? (
+  return rankedSupportingList.length > 0 ? (
```

Comment:
is this part of the fix? curious what this is for?

Reply:
no it's not but i realized profile cant be nullable in that context so was just dead code. the fix had to do with the "position: relative", cause the "position: absolute" failed unless that was there